Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -178,5 +178,257 @@ public void DoGetRequest()
         }
     }
 
-    
+    /// <summary>
+    /// <para type="synopsis">
+    /// Starts a new, asynchronous BigQuery Job. Currently, the only supported type is Query.
+    /// </para>
+    /// <para type="description">
+    /// Starts a new asynchronous job. This call requires the ‘Viewer’ role. The Type parameter 
+    /// can be -Query, -Copy, -Load, or -Extract.  Each of these job types has its own set of 
+    /// type-specific parameters to define what the job does.  Job types all share a set of 
+    /// parameters that define job attributes such as start time and handle statistics such 
+    /// as rows and raw amounts of data processed. 
+    /// Use -PollUntilComplete to have the cmdlet treat the job as a blocking operation.  
+    /// It will poll until the job has finished, and then it will return a job reference.
+    /// </para>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $job = Start-BqJob -Query "select * from book_data.classics where Year > 1900"
+    ///   </code>
+    ///   <para>Queries the classics table and returns a Job object so that results can be viewed.</para>
+    /// </example>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $job = Start-BqJob -Query "select * from classics where Year > 1900" `
+    /// -DefaultDataset $dataset -DestinationTable $table
+    ///   </code>
+    ///   <para>Queries with a default dataset and using a permanent table as the destination for results.</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs)">
+    /// [BigQuery Jobs]
+    /// </para>
+    /// </summary>
+    [Cmdlet("Start", "BqJob")]
+    public class StartBqJob : BqCmdlet
+    {
+        private class ParameterSetNames
+        {
+            public const string DoQuery = "DoQuery";
+            public const string DoCopy = "DoCopy";
+            public const string DoLoad = "DoLoad";
+            public const string DoExtract = "DoExtract";
+        }
+
+        // All-Type Parameters.
+
+        /// <summary>
+        /// <para type="description">
+        /// The project to look for jobs in. If not set via PowerShell parameter processing, it will
+        /// default to the Cloud SDK's default project.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public override string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// Turns the async call into a synchronous call by polling until the job is complete before 
+        /// returning. Can also be accessed by '-Synch'.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [Alias("Synch")]
```

Comment:
I mean in PowerShell if your parameter is -FooName and on the command line, you just type -Foo then PowerShell can still resolve it to -FooName.